style.css

```diff
@@ -21,6 +21,47 @@ li {
   -----Header-----
 */
 
+.overlay-menu {
+  display: none;
+  position: fixed;
+  z-index: 1;
+  width: 100%;
+  height: 100%;
+  color: white;
+  background-color: rgba(64, 83, 252, 0.7);
+  backdrop-filter: blur(7px);
+}
+
+.overlay-menu .content {
+  display: flex;
+  flex-direction: column;
+  gap: 30px;
+  margin: auto;
+  padding: 20px;
+  width: 80%;
```

Comment:
- Hey Team, as pointed out by the previous reviewer, this issue has not yet been resolved. what you should do is to make this width of `80%` to `100%` or better still, remove the width property so it has the default size of 100% taking up the size of its container and the design will be better to fix his changes and to improve your design.